OneOf.Serialization.Tests/OneOfJsonConverterTests.cs

```diff
@@ -21,6 +23,43 @@ public void Given_Instance_Of_OneOf_Case_When_Serializing_Then_Produces_Valid_Js
             var json = JsonConvert.SerializeObject(engine, settings);
             var engineJObject = JObject.FromObject(engine);
             json.Should().NotBeNullOrEmpty();
+            json.Should().StartWith("{").And.EndWith("}");
+            engineJObject.Should().NotBeNull();
+        }
+
+        [Fact]
+        public void Given_Enumerable_Instance_Of_OneOf_Cases_When_Serializing_Then_Produces_Valid_Json_String()
+        {
+            var engines = new List<Engine> ()
+            {
+                new WorkingEngine(Power.Create(999)),
+                new WorkingEngine(Power.Create(888)),
+            };
+            engines[1].AsT0.Status = new Status.Started(20);
+            var json = JsonConvert.SerializeObject(engines.AsEnumerable(), settings);
+            var enginesJArray = JArray.FromObject(engines.AsEnumerable());
+            json.Should().NotBeNullOrEmpty();
+            json.Should().StartWith("[").And.EndWith("]");
```

Comment:
@ra-mt what is the intention behind that? If we are dealing with JSON array?

Reply:
Yes, to verify that we get a JSON array.